benchmark/launch_experiment.py

```diff
@@ -189,11 +200,48 @@ def main():
         )
 
     else:
-        env = MORecordEpisodeStatistics(mo_gym.make(args.env_id), gamma=args.gamma)
-        eval_env = mo_gym.make(args.env_id)
+        if "mario" in args.env_id:
+            env = mo_gym.make(args.env_id, death_as_penalty=True)
```

Comment:
Can't we bundle this and time limit into the env registration?

Reply:
I think I prefer to leave this outside and let users explicitly configure these, as in Atari envs.